src/api/index.js

```diff
@@ -108,6 +108,21 @@ export async function getMentors() {
   return [];
 }
 
+export async function getFavorites() {
+  const {_id: userId} = await getCurrentUser();
+  const res = await makeApiCall(`${paths.USERS}/${userId}/favorites/`);
+  if (res.success) {
+    return res.data.mentors.map(mentor => mentor._id);
+  }
+  return [];
+}
+
+export async function addMentorToFavorites(mentorId) {
+  const {_id: userId} = await getCurrentUser();
+  const res = await makeApiCall(`${paths.USERS}/${userId}/favorites/${mentorId}`, {}, 'POST');
+  return res.success;
```

Comment:
What do you think about:
```
const {res} = await makeApiCall(`${paths.USERS}/${userId}/favorites/${mentorId}`, {}, 'POST');
return res
```

or we can have better performance with:
```
return makeApiCall(`${paths.USERS}/${userId}/favorites/${mentorId}`, {}, 'POST')
    .then({res} => res)
```

Reply:
Do you mean?

```
const {success} = await makeApiCall(`${paths.USERS}/${userId}/favorites/${mentorId}`, {}, 'POST');
return res
```

Reply:
I would left this untouched as the first attempt, dumb code again